docs/operator-manual/server-commands/argocd-application-controller.md

```diff
@@ -55,6 +55,7 @@ argocd-application-controller [flags]
       --redis-insecure-skip-tls-verify         Skip Redis server certificate validation.
       --redis-use-tls                          Use TLS when connecting to Redis. 
       --redisdb int                            Redis database.
+      --repo-error-grace int                   Grace period in seconds for ignoring consecutive errors while communicating with repo server. (default 180)
```

Comment:
Wanna add a config option via argocd-cmd-params-cm? Good for discoverability and easy configuration.

Reply:
Updated